graphql_compiler/cost_estimation/statistics.py

```diff
@@ -0,0 +1,71 @@
+# Copyright 2019-present Kensho Technologies, LLC.
+from abc import ABCMeta, abstractmethod
+
+from frozendict import frozendict
+import six
+
+
+@six.python_2_unicode_compatible
+@six.add_metaclass(ABCMeta)
+class Statistics(object):
+    """Abstract class for statistics regarding GraphQL objects.
```

Comment:
I don't think `regarding GraphQL objects` is clear. It seems to me that it has something to do with python objects from the `graphql-core` library.

The statistics we gather are all about `the current state of a backend instance`. That's how I'd define it. Thoughts?

Reply:
It's not ideal, since we need statistics about every class in the schema, not about a specific backend. I don't care if half the backend is SQL and the other half is something else, I need statistics about all databases in one Statistics object.

Reply:
Decided on ```Abstract class for statistics regarding the data in a backend instance.```